go-controller/pkg/ovn/logical_switch_cache.go

```diff
@@ -0,0 +1,165 @@
+package ovn
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+	"sync"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	"k8s.io/klog"
+	ipam "k8s.io/kubernetes/pkg/registry/core/service/ipallocator"
+)
+
+// logicalSwitchInfo contains information corresponding to the node. It holds the
+// subnet allocations (v4 and v6) as well as the IPAM allocator instances for each
+// subnet managed for this node
+// logicalSwitchInfo is used to save switch specific information such as subnets,
+// allocated IPs per switch etc.
+type logicalSwitchInfo struct {
```

Comment:
L18-19 captures somewhat similar info at L15-17, maybe you can remove those lines or merge them.

Reply:
done.